libc/src/string/memory_utils/generic/builtin.h

```diff
@@ -10,16 +10,15 @@
 #define LLVM_LIBC_SRC_STRING_MEMORY_UTILS_GENERIC_BUILTIN_H
 
 #include "src/__support/macros/attributes.h" // LIBC_INLINE
-#include "src/__support/macros/config.h"     // LIBC_HAS_BUILTIN
 #include "src/string/memory_utils/utils.h"   // Ptr, CPtr
 
 #include <stddef.h> // size_t
 
 namespace LIBC_NAMESPACE {
 
-static_assert(LIBC_HAS_BUILTIN(__builtin_memcpy), "Builtin not defined");
-static_assert(LIBC_HAS_BUILTIN(__builtin_memset), "Builtin not defined");
-static_assert(LIBC_HAS_BUILTIN(__builtin_memmove), "Builtin not defined");
+static_assert(__has_builtin(__builtin_memcpy), "Builtin not defined");
+static_assert(__has_builtin(__builtin_memset), "Builtin not defined");
+static_assert(__has_builtin(__builtin_memmove), "Builtin not defined");
```

Comment:
Use the same pattern described above using `#error`.

Reply:
Done.